src/libraries/System.Security.Cryptography/tests/HashDerivedTests.cs

```diff
@@ -11,36 +11,57 @@ public static class HashDerivedTests
         public static void HashSize_SetForDerived_SHA1()
         {
             using DerivedSHA1 sha = new DerivedSHA1();
-            Assert.Equal(160, sha.HashSize);
+            Assert.Equal(SHA1.HashSizeInBits, sha.HashSize);
```

Comment:
I assume there's a test somewhere that asserts SHA1.HashSizeInBits is 160?

Reply:
No there weren't. So I added some that do it in a more direct manner.